lib/biokbase/Transform/script_utils.py

```diff
@@ -136,7 +136,7 @@ def parse_docs(docstring=None):
     remainder = script_details["Args"]
     argument_values = list()
     for k in reversed(argument_keys):
-        remainder, value = remainder.split(k)        
+        remainder, _, value = remainder.rpartition(k)       
```

Comment:
This is needed because one of our fields is named `name`, which is what caused the conflict leading to the other PR.